spring-boot-admin-server-ui/package.json

```diff
@@ -14,6 +14,7 @@
     "@fortawesome/free-regular-svg-icons": "^5.8.1",
     "@fortawesome/free-solid-svg-icons": "^5.8.1",
     "@fortawesome/vue-fontawesome": "^0.1.6",
+    "@panter/vue-i18next": "^0.15.0",
```

Comment:
Are there any reasons why you've chosen this over vue-i18n? Is it because it uses  i18next and doesn't reimplement that stuff?

Reply:
Nope, just found i18next useful but I will switch over to vue-18n :)

Reply:
Please don’t - I was just curious